portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that gets data from a csv and parses it to display in a chart.*/
+@WebServlet("/chart-data")
+public class ChartServlet extends HttpServlet {
+
+  // Ordered map to store the pairs of values parsed from the csv.
```

Comment:
I think this comment got lost in the last PR, so I'll rewrite it here: 'make sure the comment explains more what the purpose of this variable is! So the 'why', instead of the 'what'. For instance, "this map stores search trends for the search term 'meme' keyed by year.", or something like that!'

Reply:
Done!